### DISCUSSION TITLE
C++ Extension Debugger and CUDA

### DISCUSSION THREAD

**Discussion by browntarik** [microsoft/vscode-cpptools#11029] [General]
Quoting user @pburbaker (https://github.com/pbrubaker) here, this was a question that was posed on reddit and want to make sure it gets answered - "I actually had a question about how debugging works. I work on a language and compiler ([ISPC](http://ispc.github.io/)) and we currently have a language extension/LSP. The artifacts ISPC produces are headers included in C/C++ files and object files that are linked by the linker. Try as I might, I can't get debugging to work with the C/C++ extension. I noticed the extension references CUDA, does the same need to happen for our language?"

**Comment by browntarik** [comment]
@WardenGnaw You might have an answer for this?

**Reply by sean-mcmanus** [reply]
@pburbaker Yeah, see https://github.com/microsoft/vscode-cpptools/pull/10476 for an example.

**Comment by pbrubaker** [comment]
Hey Tarik!  Thanks for following up with this, I really appreciate it.  I'm going to add @ethandav to this thread as he is going to be working on this.  @sean-mcmanus we'll take a look at that, thanks a bunch!
